cypress/integration/computation_menu.spec.ts

```diff
@@ -0,0 +1,116 @@
+
+            yearsWithModernElectionSystem.forEach((year) => {
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                cy.get("#merge_to_11_districts_checkbox").check();
+                cy.get("#merge_to_11_districts_checkbox").should("be.checked");
+            });
+        });
+    });
+
+    context("use 2021 seat distribution checkbox", () => {
+        it("works for relevant years", () => {
+            cy.visit("").wait(1000);
+
+            yearsWithModernElectionSystem.forEach((year) => {
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                cy.get("#2021_distribution_checkbox").check();
+                cy.get("#2021_distribution_checkbox").should("be.checked");
+            });
+        });
+    });
+
+    context("merge checkbox, then 2021 distribution checkbox", () => {
+        it("works for relevant years", () => {
+            cy.visit("").wait(1000);
+
+            yearsWithModernElectionSystem.forEach((year) => {
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                cy.get("#merge_to_11_districts_checkbox").check();
+                cy.get("#2021_distribution_checkbox").check();
+                cy.get("#merge_to_11_districts_checkbox").should("be.checked");
+                cy.get("#2021_distribution_checkbox").should("be.checked");
+            });
+        });
+    });
+
+    context("2021 distribution checkbox, then merge checkbox", () => {
+        it("works for relevant years", () => {
+            cy.visit("").wait(1000);
+
+            yearsWithModernElectionSystem.forEach((year) => {
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                cy.get("#2021_distribution_checkbox").check();
+                cy.get("#merge_to_11_districts_checkbox").check();
+                cy.get("#merge_to_11_districts_checkbox").should("be.checked");
+                cy.get("#2021_distribution_checkbox").should("be.checked");
+            });
+        });
+    });
+
+    context("algorithms", () => {
+        years.forEach((year) => {
+            it(`works for ${year}`, () => {
+                cy.visit("").wait(1000);
+                cy.get("#year_select").select(`${year}`);
+                cy.get("#year_select").should("have.value", `${year}`);
+                const algorithms = [
+                    "SAINTE_LAGUE",
+                    "D_HONDT",
+                    "LARGEST_FRACTION_HARE",
+                    "LARGEST_FRACTION_DROOP",
+                    "LARGEST_FRACTION_HAGENBACH_BISCHOFF",
+                ];
+                algorithms.forEach((algorithm) => {
+                    cy.get("#algorithm_select").select(algorithm);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['cy' is not defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5611900)